DataFormats/Detectors/TRD/include/DataFormatsTRD/Constants.h

```diff
@@ -56,6 +56,7 @@ constexpr float GRANULARITYTRKLSLOPE = 1.f / PADGRANULARITYTRKLSLOPE; // granula
 
 // OS: Should this not be flexible for example in case of Kr calib?
 constexpr int TIMEBINS = 30; // the number of time bins
+constexpr int NBINSANGLEDIFF = 26; // the number of bins for the track angle used for the vDrift and ExB calibration based on the tracking (last bin is for under-/overflow entries)
 
 // Trigger parameters
```

Comment:
These two sections should be outside `Constants.h`. None of them are actually constants but configurable parameters, maybe in Simulation of Common parameters? One could pass the number of time bins through the steering code, then the trigger parameters are calculated based on that. So, yet another to-do.